pkgs/misc/emulators/maiko/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub, cmake, libX11 }:
+
+stdenv.mkDerivation rec {
+  pname = "maiko";
+  version = "2021-04-14";
```

unstable- prefix for non-releases?

---

I don't think that the developers intend to cut releases so adding unstable would be misleading.

---

```suggestion
  version = "unstable-2021-04-14";
```
Still we are fetching a non release version.